### PR TITLE
fix: check user existance on assigning and revoking

### PR DESCRIPTION
**Pull request by moogacs**
### Review checklist

- [ ] Documentation has been updated, if necessary. Link to changed documentation:
- [ ] Chaos pipeline run or not necessary. Link to pipeline:
- [ ] All new code is covered by tests where it is reasonable.
- [ ] Performance tests have been run or not necessary.

### CODE REVIEW

File: test/acceptance/authz/rbac_auto_no_permissions_test.go [hunk 1]
```diff
@@ -125,7 +125,7 @@ func TestAuthzAllEndpointsNoPermissionDynamically(t *testing.T) {
 			if slices.Contains(ignoreEndpoints, endpoint.path) {
 				return
 			}
-
+			t.Log(resp.Body)
```
Comment (dirkkul) [inline test/acceptance/authz/rbac_auto_no_permissions_test.go]: debug leftover? :)

File: adapters/handlers/rest/authz/handlers_authz.go [hunk 2]
```diff
@@ -427,6 +443,12 @@ func (h *authZHandlers) revokeRole(params authz.RevokeRoleParams, principal *mod
 		return authz.NewRevokeRoleBadRequest().WithPayload(cerrors.ErrPayloadFromSingleErr(fmt.Errorf("user id can not be empty")))
 	}
 
+	for _, role := range params.Body.Roles {
+		if strings.TrimSpace(role) == "" {
+			return authz.NewRevokeRoleBadRequest().WithPayload(cerrors.ErrPayloadFromSingleErr(fmt.Errorf("one or more of the roles you want to revoke is empty")))
+		}
+	}
```
Comment (dirkkul) [inline adapters/handlers/rest/authz/handlers_authz.go:450]: Is this needed? Wouldn't we just end up with "user not found" if I pass in " "? Same for the other endpoint

Comment (moogacs) [reply adapters/handlers/rest/authz/handlers_authz.go:450]: That is for the passed roles to revoke or assign

File: test/acceptance/authz/rbac_auto_no_permissions_test.go (hunk 1 above)
Comment (moogacs) [reply test/acceptance/authz/rbac_auto_no_permissions_test.go]: 🤦‍♂️ i thought i removed, good catch on it

File: adapters/handlers/rest/authz/handlers_authz.go (hunk 2 above)
Comment (dirkkul) [reply adapters/handlers/rest/authz/handlers_authz.go:450]: oki - but wouldn't it be the same there? We return that the role " " does not exist?

Comment (moogacs) [reply adapters/handlers/rest/authz/handlers_authz.go:450]: " " is empty (invalid request body) , doesn't exists it requires authZ t

**Review by dirkkul — APPROVED** [review]

**Pull request merged**